.github/workflows/tests.yml

```diff
@@ -15,13 +15,13 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-        ruby-version: [2.6, 2.7]
+        ruby-version: [2.6, 2.7, '3.0', 3.1]
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Set up Ruby
-        uses: ruby/setup-ruby@a699edbce608a2c128dedad88e3b6a0e28687b3c
+        uses: ruby/setup-ruby@v1
```

Comment:
Could you revert this please (or even better pin it to the latest sha: [180e1d1f1c5eadbfbab3dab6fb79ab0c07e3cecc](https://github.com/ruby/setup-ruby/commit/180e1d1f1c5eadbfbab3dab6fb79ab0c07e3cecc))

Pinning to tags is generally unsafe as someone in the future could take control of that repo, republish a different tag with the same version, and then we'd be running malicious code. Git sha's guarantee that you're running the code that you actually intended to, which is why I use them.

Reply:
Done. Good point on the SHA vs version. FWIW - You'll need to bump this version as you bring newer rubies into the matrix.